app/src/main/java/com/willowtree/vocable/AppKoinModule.kt

```diff
@@ -31,6 +25,14 @@ import org.koin.dsl.module
 object AppKoinModule {
 
     fun getModule() = module {
+        // Activity Scope
```

Comment:
Little bit of a nit, but does this comment add any context the line immediately beneath it does not?

Reply:
Nope this is valid.